src/Random/MersenneTwister.php

```diff
@@ -43,27 +43,27 @@ public function seed($seed)
             $this->mt[$i] = ($this->f * (
                 $this->mt[$i - 1] ^ (($this->mt[$i - 1] >> ($this->w - 2)) & 0b11)
             ) + $i) & $this->wMask;
-            assert('$this->mt[$i] <= $this->wMask');
+            assert($this->mt[$i] <= $this->wMask);
         }
-        assert('count($this->mt) === 624');
+        assert(count($this->mt) === 624);
```

Comment:
I think this has been corrected in `master` so will probably merge it in here before attempting to merge towards it.

Reply:
will rebase the branch when fixing the issues you found, so that shouldn't be an issue anymore.